src/services/offer-service.ts

```diff
@@ -0,0 +1,7 @@
+import { URLs } from '~/constants/request'
+import { CreateOfferData } from '~/containers/offer-page/create-offer/CreateOffer'
+import { axiosClient } from '~/plugins/axiosClient'
+
+export const OfferService = {
+  createOffer: async (data:CreateOfferData) => await axiosClient.post(URLs.offers.create, data)
```

Comment:
i guess, you should add Promise<AxiosResponse<yourType>> as return type of createOffer

Reply:
fixed